src/test/resources/application.properties

```diff
@@ -1,5 +1,8 @@
 spring.datasource.url=jdbc:h2:mem:bookstoreDB
 spring.datasource.driverClassName=org.h2.Driver
-spring.datasource.username=
-spring.datasource.password=
+spring.datasource.username=sa
+spring.datasource.password=password

```

Comment:
What is the reason for adding this?

Reply:
Test were failed so added it just in case